portfolio/src/main/webapp/aboutme.html

```diff
@@ -6,22 +6,33 @@
     <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1 id="page-title">My Portfolio</h1>
-    <div id="navbar">
-      <ul>
-        <li><a href="index.html">Home</a></li>
-        <li><a href="projects.html">Projects</a></li>
-        <li><a href="gallery.html">Gallery</a></li>
-        <li><a href="blog.html">Blog</a></li>
-        <li><a class="active" href="aboutme.html">About Me</a></li>
-      </ul>
-    </div>
+    <h1 id="site-title">My Portfolio</h1>
+    <!-- navbar -->
+    <ul>
+      <li><a href="index.html">Home</a></li>
+      <li><a href="projects.html">Projects</a></li>
+      <li><a href="gallery.html">Gallery</a></li>
+      <li><a href="blog.html">Blog</a></li>
+      <li><a class="active" href="aboutme.html">About Me</a></li>
+    </ul>
 
     <div id="content">
-      <h1>About Me</h1>
-      <p>My name is William Lew. I am from San Ramon, CA. I go to Lewis & Clark College and am a Computer Science major.
-          My hobbies include playing video games, writing, listening to music, and coding.</p>
-      <img src="/images/me.jpg" alt="A photo of me" width="734.7" height="489.75">
+      <div class="page-title" id="about-me">
+        <h1>About Me</h1>
+      </div>
+      <div id="about-me-info">
+        <p>My name is William Lew. I am from San Ramon, CA. I go to Lewis & Clark College and am a Computer Science major.
+            My hobbies include playing video games, writing, listening to music, and coding.</p>
+        <br>
+        <img src="/images/me.jpg" alt="A photo of me">
+        <br>
+        <br>
+        <div id="logos">
+          <a href="https://github.com/0tpyrk"><img src="images/logos/github.png" alt="GitHub"></a>
+          <a href="https://www.linkedin.com/in/william-lew-382018189/"><img src="images/logos/linkedin.png" alt="Linkedin"></a>
```

Comment:
Please avoid lines longer than 80 characters

Reply:
Will do, thanks for the reminder.